src/vsr/client.zig

```diff
@@ -513,7 +516,7 @@ pub fn Client(comptime StateMachine_: type, comptime MessageBus: type) type {
         }
 
         /// Releases a message back to the message bus.
-        pub fn release(self: *Self, message: *Message) void {
+        pub fn release_message(self: *Self, message: *Message) void {
```

Comment:
I prefer `release_message` to `release` even ignoring the name collision -- it mirrors `get_message`.